catalog/app/containers/Bucket/Listing.tsx

```diff
@@ -1152,6 +1154,18 @@ export function Listing({
         },
       })
     }
+    columnsWithValues.push({
+      field: 'actions',
+      headerName: '',
+      align: 'right',
+      width: 0,
+      renderCell: (params: DG.GridCellParams) =>
+        params.id === '..' ? (
+          <div />
```

Comment:
RowActions returns null when there's no actions to show, so i'd suggest returning null here as well

Reply:
`null` conflicts with type annotations, I can return `<></>` in both cases

Reply:
`null` feels more appropriate, but whatever